### ISSUE TITLE
Add support for ICC color profiles to saved PNG

### ISSUE THREAD

**Issue by Beep6581** [Beep6581/RawTherapee#2598]
Originally reported on Google Code with ID 2615

```
I tested tip 4.2.45 e13f96d51b9b and neither saved 8-bit nor 16-bit PNG files contain
ICC color profiles when one was chosen in RT.
```

Reported by `entertheyoni` on 2014-12-22 19:39:43

**Comment by Beep6581** [comment]
It seems RT now includes an embedded ICC in the saved PNG files.

Someone else please before closing.

**Comment by agriggio** [comment]
Confirmed, implemented in 527f41c2541414cf443bd3460065111224769356
